apps/consumer-server/package.json

```diff
@@ -4,12 +4,12 @@
   "license": "GPL-3.0-or-later",
   "private": true,
   "scripts": {
-    "dev": "NODE_OPTIONS=--max_old_space_size=4096 NODE_ENV=development ts-node-dev src/main.ts",
+    "dev": "NODE_OPTIONS=--max_old_space_size=4096 NODE_ENV=development ts-node-dev --respawn --watch=.env,.env.local src/main.ts",
```

Comment:
Here and elsewhere, I'm not sure you need to watch `.env.local` (as I don't think that file is used by anything), but watching `.env` seems like a good idea to me. I see that it is probably used in *some* of our projects, but not all. The reason I think it's important to get this precisely right is that otherwise it may be misleading to developers trying to understand how this project works.